VERSION

```diff
@@ -1 +1 @@
-DOCKER_BUILD_TAG=0.9.1
+DOCKER_BUILD_TAG=3.0.1-rc1
```

Comment:
I can't fully agree with this VERSION file approach. Imho, git tag is much more reliable and obvious source of truth.  As long as you have both, you will have inconsistencies between them. Take a look at this:

1. We take git tag as a source of truth: https://github.com/KillrVideo/killrvideo-dse-config/blob/master/.travis.yml#L15
2. We take hardcoded data in a file as a source of truth https://github.com/KillrVideo/killrvideo-dse-config/blob/1a31a70883597d98b4f90a86986f782d5e3319b4/build/docker-build.sh#L5

It's just a question of time when someone (like me) will forget to update the version file and - ooops - we have a wrong image.